modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -102,7 +102,7 @@ const CreationMenu = ({
 		let currentItemCount = 0;
 
 		return (
-			<ClayDropDown.ItemList>
+			<ClayDropDown.ItemList className="dropdown-menu-indicator-start">
```

Comment:
We should add this class only when icon is set. Always adding this class introduces a bug:
![screen](https://user-images.githubusercontent.com/5435511/108357015-2cef4c00-71ed-11eb-8667-52447fa4aafe.png)

Reply:
Yes, should have seen that :+1: 
As far as I remember we don't have an easy way to see if icons are going to be used but it shouldn't be hard to add.